centreon/cypress/scripts/cypress-cli.sh

```
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-docker run --shm-size 4gb -v "$PWD:/tmp" -w "/tmp" cypress/included:13.6.2 --component --browser=chrome $@
\ No newline at end of file
+docker run -it -v $PWD:/tmp -w /tmp --name=cypress cypress/included:14.1.0 --component --browser electron $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)